pandas/tests/util/test_validate_fillna_kwargs.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from pandas.util._validators import validate_fillna_kwargs
+
+
+def test_no_value_or_method():
```

Comment:
Was this not tested before?

Reply:
I couldn't find any mention of this specific function in any test files after doing a search for it.  There may be higher level tests that check the behaviour of the fillna method for this though.